packages/teams-js/test/private/files.spec.ts

```diff
@@ -483,40 +483,67 @@ describe('files', () => {
     it('should not allow calls before initialization', () => {
       expect(() => files.getFileDownloads(emptyCallback)).toThrowError('The library has not yet been initialized');
     });
+    describe('v1', () => {
+      it('should not allow calls without frame context initialization', async () => {
+        await utils.initializeWithContext('settings');
+        expect(() => files.getFileDownloads(emptyCallback)).toThrowError(
+          'This call is only allowed in following contexts: ["content"]. Current context: "settings"',
+        );
+      });
 
-    it('should not allow calls without frame context initialization', async () => {
-      await utils.initializeWithContext('settings');
-      expect(() => files.getFileDownloads(emptyCallback)).toThrowError(
-        'This call is only allowed in following contexts: ["content"]. Current context: "settings"',
-      );
-    });
-
-    it('should not allow calls with empty callback', async () => {
-      await utils.initializeWithContext('content');
-      expect(() => files.getFileDownloads(null)).toThrowError();
+      it('should trigger callback correctly', async () => {
+        expect.assertions(4);
+        await utils.initializeWithContext('content');
+        const mockFileDownloads: files.IFileItem[] = [
+          {
+            timestamp: new Date(),
+            title: 'title',
+            extension: 'docx',
+          },
+        ];
+        let called = false;
+
+        const callback = jest.fn((err, fileList) => {
+          called = true;
+
+          expect(err).toBeFalsy();
+          expect(fileList).toEqual(mockFileDownloads);
+          expect(called).toBe(true);
```

Comment:
Should this line be outside the callback since `expect(called).toBe(true)` will always be true in the callback?

Reply:
Boof github still has a duplicate comment bug

Reply:
Boof github still has a duplicate comment bug

Reply:
technically, I don't even need to test if it's called because i'm already doing that with the expect.assertions(which I know if technically for catch blocks).  If I remove the call to the callback, the test will fail saying there is only one assertion

Reply:
Hmm maybe we don't even need called then? And have just three assertions

Reply:
so I understand that it looks redundant, but it actually isn't because the assertions within the callback won't ever run if the callback isn't called.  If we set a variable and changed it outside of any function, this would be useless, but because the value never gets 

We can't check if called is true outside because the callback function is actually called AFTER the test is run...just to show this, I added a bunch of console statements to illustrate this

![image](https://user-images.githubusercontent.com/9360417/158701384-491f857b-9372-4a97-bc8c-3168250ff00d.png)